README.md

```diff
@@ -89,7 +89,10 @@ limited by the capacity of a single physical machine, lest we sacrifice the
 property of complete auditability.  The Lightning Network [\[7\]][7] can help
 scale Bitcoin transaction volume by leaving some transactions off the ledger
 completely and is well suited for micropayments and privacy preserving payment
-rails, but may not be suitable for more generalized scaling needs.
+rails, and it is suitable for more generalized scaling needs.
+<!---
+state-channels allow for generalized smart contracts in the lightning network
+--->
```

Comment:
Yes but ultimately it's about tokens, yes?  Like, a state-channel is an impermanent thing, and the difference between before-and-after a state channel is that some tokens had moved.  So ultimately it's only about token transfers, within the same ledger (though it can happen atomically with such transfers in one or more other ledgers, it's still not what we call "asymmetric token transfer")